app/code/Magento/Sales/Block/Order/OrderItemsPager.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Sales\Block\Order;
+
+use Magento\Framework\Data\Collection;
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Html pager block
+ *
+ * @SuppressWarnings(PHPMD.ExcessivePublicCount)
+ * @api
+ * @since 100.0.2
+ */
+class OrderItemsPager extends \Magento\Catalog\Block\Product\Widget\Html\Pager
+{
+    /**
+     * @var int
+     */
+    protected $_total_item_count;
+
+    /**
+     * Return current page
+     */
+    public function getCurrentPage()
+    {
+        if (is_object($this->_collection)) {
+            return $this->_collection->getCurPage();
+        }
+        return (int)$this->getRequest()->getParam($this->getPageVarName(), 1);
+    }
+
+    /**
+     * Set collection for pagination
+     *
+     * @param Collection $collection
+     * @return OrderItemsPager
+     */
+    public function setCollection($collection)
```

Comment:
This method looks exactly like the one from the parent class. I believe we can remove it from here.

Reply:
Has been removed.